posts/2020-05-15-six-years-of-rust.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Six Years of Rust"
+author: The Rust Core Team
```

Comment:
I would probably say "The Rust Team" here

Reply:
It's been attributed to the core team in previous years, but I have no opinion myself.

Reply:
The core team didn't write this post, and so we shouldn't be named as the authors.